lang/macro/src/codegen/events.rs

```diff
@@ -135,7 +135,7 @@ impl EventHelpers<'_> {
             .collect::<Vec<_>>();
 
         quote! {
-            #[derive(scale::Encode)]
+            #[derive(scale::Encode, scale::Decode)]
```

Comment:
Why do we need `Decode` here?

Reply:
For the test to decode the Event. I could add it under `test-env` but it should get pruned anyway by the build if not used.